src/main/java/reactor/kafka/receiver/internals/ConsumerEventLoop.java

```diff
@@ -164,6 +164,11 @@ private void onPartitionsRevoked(Collection<TopicPartition> partitions) {
             });
     }
 
+    @Override
+    public boolean onEmitFailure(SignalType signalType, Sinks.Emission emission) {
+        return isActive.get();
```

Comment:
the parkNanos was removed, could be conditionally added back while also checking the `signalType`? this effectively can loop indefinitely in case of `TERMINATED` for instance, which would be wasteful

Reply:
Nice catch!